packages/data-service/src/instance-detail-helper.ts

```diff
@@ -88,6 +89,7 @@ export type InstanceDetails = {
   genuineMongoDB: GenuineMongoDBDetails;
   dataLake: DataLakeDetails;
   featureCompatibilityVersion: string | null;
+  isAtlas: boolean;
```

Comment:
should this be used here? https://github.com/mongodb-js/compass/pull/2589/files#diff-0a7ade426ce694c5c7bd013c0f5f85f445376c0210d589cc86c704bf9db5e3f9R27

only after the connection happens?